home_work_14/index.js

```diff
@@ -0,0 +1,142 @@
+//1
+
+const outputCentury = document.querySelector("#result-calc-century");
+const inputYear = document.querySelector("#year");
+const buttonCalculateCentury = document.querySelector("#calc-century");
+
+const getCentury = () => {
+  let cutback;
+  if (inputYear.value && inputYear.value > 0) {
+    const century = 1 + Math.trunc(inputYear.value / 100);
+    if (century > 10 && [11, 12, 13, 14].includes(century % 100)) {
+      cutback = "th";
+    } else if (century % 10 === 1) {
+      cutback = "st";
+    } else if (century % 10 === 2) {
+      cutback = "nd";
+    } else if (century % 10 === 3) {
+      cutback = "rd";
+    } else {
+      cutback = "th";
+    }
+    outputCentury.innerHTML = `${century}${cutback}`;
+  } else {
+    outputCentury.innerHTML = "Enter year";
+  }
+};
+buttonCalculateCentury.addEventListener("click", getCentury);
+
+//4
+
+const outputMinMax = document.querySelector("#result-calc-min-max");
+const inputNumbers = document.querySelector("#min-max");
+const buttonCalculateMinMax = document.querySelector("#calc-min-max");
+
+const getMinMax = () => {
+  if (inputNumbers.value) {
+    const arrayNumbers = inputNumbers.value.split(" ");
+    const resultValues = arrayNumbers
+      .filter((el) => el !== "" && !isNaN(el))
+      .sort((el1, el2) => +el2 - +el1);
```

Comment:
Try to think of how you can optimise your algorithm using only 1 method. For example `.reduce`